openlibrary/templates/account/create.html

```diff
@@ -28,12 +28,16 @@ <h1>$_("Sign Up")</h1>
                 <span class="smaller lighter">$input.help</span>
         </div>
         <div class="input">
+            <script src="https://kit.fontawesome.com/a076d05399.js"></script>
```

Comment:
```suggestion
```
This script tag is being added to the page each time that `field` is called in this template.  Anyway, we should save a copy of each visibility icon in our images directory.